src/TxHistory/TxDetails/TxDetails.tsx

```diff
@@ -139,7 +139,7 @@ export const TxDetails = () => {
         </View>
       </Screen>
 
-      {addressDetail && (
+      {!!addressDetail && (
```

Comment:
how does this change anything?

Reply:
Error: Text strings must be rendered within a component.